tests/test_2dm.cpp

```diff
@@ -270,6 +271,98 @@ TEST( Mesh2DMTest, Basement3CellElevationTest )
   MDAL_CloseMesh( m );
 }
 
+
+
+TEST( Mesh2DMTest, SaveMeshToFile )
```

Comment:
can you please add test where you want to store file with > 4 vertices per face?

Reply:
Do you know if there is some file available with > 4 vertices per face in the test data ?

Reply:
I have added a test with a fake MemoryMesh with maximum vertices per face equal to 5.

Reply:
MemoryMesh is not exported from the mdal library, so you cannot use it (it is implementation detail). if you can just remove this test for the time being

Reply:
done